src/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
     <title>Leaderboard</title>
 </head>
```

Comment:
- [ ] Well done for following most of the layout guide, however, the general layout is still off because of the huge space on the right which makes the layout look unbalanced. You could consider centering the content of the div, then using width and margins to control how much space each child div takes up in order to achieve the desired layout.
Please see attached image below to spot the difference in your layout compared to what is actually required 👇 
### Wireframe Layout
![image](https://user-images.githubusercontent.com/46839548/209453488-f076d3a6-fe7d-4f84-a9f6-de508e10646d.png)

### Your Layout
![image](https://user-images.githubusercontent.com/46839548/209453448-79e0ba67-9fd5-4756-8ea3-fc1e7c786fcf.png)